Source/Scene/Cesium3DTileStyle.js

```diff
@@ -246,13 +246,24 @@ define([
                 return this._show;
             },
             set : function(value) {
+                if (typeof value === 'boolean') {
+                    this._show = new Expression(String(value));
+                } else if (typeof value === 'string') {
+                    this._show = new Expression(value);
+                } else if (defined(value.conditions)) {
+                    this._show = new ConditionsExpression(value);
+                } else if (defined(value.expression) || defined(value.conditionsExpression)) {
+                    this._show = value;
+                } else {
+                    this._show = undefined;
+                    return;
+                }
                 this._showShaderFunctionReady = false;
-                this._show = value;
             }
         },
```

Comment:
It works, but I don't like code design here, because it is not really clear why one time `getColorShaderFunction` returns `undefined` and other times a valid shader, if both times value of `color` is equal to `DEFAULT_JSON_COLOR_EXPRESSION`.

So I would prefer: In case no color expression is defined `color` returns `undefined` instead of `DEFAULT_JSON_COLOR_EXPRESSION`. In `Cesium3DTileBatchTable.prototype.applyStyle` change code from:
```
            var color = style.color.evaluateColor(frameState, feature, scratchColor);
            var show = style.show.evaluate(frameState, feature);
```
to
```
            var color = defined(style.color) ? style.color.evaluateColor(frameState, feature, scratchColor) : 'white';
            var show = defined(style.show) ? style.show.evaluate(frameState, feature) : true;
```